apps/chat/src/components/Sidebar/Sidebar.tsx

```diff
@@ -105,7 +105,7 @@ const Sidebar = <T,>({
 
   const resizeTriggerColor = classNames(
     'xl:bg-accent-primary xl:text-accent-primary',
-    isResizing ? 'bg-accent-primary text-accent-primary' : '',
+    isResizing ? 'md:bg-accent-primary md:text-accent-primary' : '',
```

Comment:
instead of `md:` I suggest to set `bg-layer-3 text-secondary` only if `!isResizing`
![image](https://github.com/user-attachments/assets/532fc60c-b678-4794-a649-74a3a82c2dbe)

Reply:
Code changed as suggested.
Thank you for the advice!